core/src/main/kotlin/org/evomaster/core/problem/rest/RestActionBuilderV3.kt

```diff
@@ -728,4 +723,14 @@ object RestActionBuilderV3 {
         }
     }
 
+    fun getBasePathFromURL(serverUrl: String): String {
```

Comment:
this should take as input the `swagger` object, and not the `serverUrl`.
point is that choosing `.servers[0]` is now an hack, as in the future might want to support other indices as well